example/src/main/tut/data/Free.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: docs
+title: "Free"
+---
+
+# Free
+
+Free is a recursive data structure that either returns a vale `A` or returns an effect computation in `F[_]` that yields
+another `Free`. Free can be summarized with following signature:
+
+`type Free[F[_], A] = A \/ F[Free[F,A]]`
+
+Free gets it's namesake from it's Monad instance which can be used to derive a monad from any arbitrary `Functor`.
```

Comment:
"gets its name from its"...

"any" and "arbitrary" are redundant together.

Maybe add a note that this _doesn't_ have the built-in `Coyoneda` that 7.x and cats have.